stories/atoms/all-form-controls.js

```diff
@@ -3,52 +3,84 @@ import '../../src/atoms/cc-input-number.js';
 import '../../src/atoms/cc-input-text.js';
 import '../../src/atoms/cc-select.js';
 import '../../src/atoms/cc-toggle.js';
+import { defaultThemeStyles } from '../../src/styles/default-theme.js';
 import { makeStory } from '../lib/make-story.js';
 
 const conf = {
-  css: `
-    h1 {
-      font-size: 1.2em;
-      margin-bottom: 1em;
-    }
-
-    h1+div {
-      align-items: flex-end;
-      border-bottom: solid 1px gray;
-      display: flex;
-      gap: 1em;
-      margin-bottom: 2em;
-      padding-bottom: 2em;
-    }
-  `,
+  css: [
+    defaultThemeStyles,
```

Comment:
The `css` used by `makeStory` expects a string, it is directly used under the hood to create a `<style>` tag. It doesn't work like lit so the array doesn't work and the result of lit's `css` function doesn't work either.

Reply:
```
defaultThemeStyles.toString(),
```

```
[
/* ... */
].join('\n'),
```